### DISCUSSION TITLE
upload 组件 多文件上传 fileList里面是累加的，并不是当前的上传文件列表

### DISCUSSION THREAD

**Discussion by zhaoPengInGithub** [ant-design/ant-design#31398] [Q&A]
![image](https://user-images.githubusercontent.com/29244126/125558807-aa4e9009-2dd4-4038-9b93-d5c98c7e933c.png)
每次上传2个文件，但是第二次上传的时候第一次上传的文件也在，看upload的api里面是写的fileList里面是本次上传的文件，但实际情况不是，这种怎么拿到这次的文件列表

**Comment by ZHHHuaZhang** [comment]
我现在发现我把文件传入fileList 再次改变fileList 的时候也是累加的
